File: pennyway-socket/src/main/java/kr/co/pennyway/socket/PennywaySocketApplication.java
package kr.co.pennyway.socket;

import jakarta.annotation.PostConstruct;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

import java.util.TimeZone;

@SpringBootApplication
public class PennywaySocketApplication {
    public static void main(String[] args) {
        SpringApplication.run(PennywaySocketApplication.class, args);
    }

    @PostConstruct
    public void init() {
        TimeZone.setDefault(TimeZone.getTimeZone("Asia/Seoul"));
    }
}

File: pennyway-socket/src/main/java/kr/co/pennyway/PennywaySocketApplication.kt
package kr.co.pennyway;

import jakarta.annotation.PostConstruct
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import java.util.*

@SpringBootApplication
class PennywaySocketApplication {
    @PostConstruct
    fun setDefaultTimeZone() {
        TimeZone.setDefault(TimeZone.getTimeZone("Asia/Seoul"))
    }
}

fun main(args: Array<String>) {
    runApplication<PennywaySocketApplication>(*args)
}
